apps/laboratory/src/components/Ethers/EthersModalInfo.tsx

```diff
@@ -0,0 +1,10 @@
+import * as React from 'react'
+import { useWeb3ModalAccount } from '@web3modal/ethers/react'
+
+import { Web3ModalInfo } from '../Web3ModalInfo'
+
+export function EthersModalInfo() {
+  const { isConnected, address, chainId } = useWeb3ModalAccount()
```

Comment:
This will probably cause some hydration issues. Since we don't fully support SSR, the `isConnected` value will be different between the server and the client.

Instead, we could:
-  Force it to use client-side rendering: https://nextjs.org/docs/messages/react-hydration-error#possible-ways-to-fix-it
- Or render the `<Web3ModalInfo/>` component even though we don't have connection yet but there wouldn't be any info inside.

Reply:
added a ready check 🖖